src/main/java/org/javawebstack/orm/query/QueryCondition.java

```diff
@@ -1,23 +1,41 @@
 package org.javawebstack.orm.query;
 
-import org.javawebstack.orm.TableInfo;
+import org.javawebstack.orm.exception.ORMQueryException;
 
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
+import java.util.Locale;
 
 public class QueryCondition implements QueryElement {
 
+    private static final List<String> VALID_OPERATORS = Arrays.asList(
+            "=",
+            "<=>",
+            "!=",
+            "<>",
+            "<=",
+            ">=",
+            "<",
+            ">",
+            "is null",
+            "is not null",
+            "is",
+            "is not",
+            "in",
+            "not in",
+            "like",
+            "not like"
+    );
+
     private final Object left;
     private final String operator;
     private final Object right;
     private final boolean not;
 
     public QueryCondition(Object left, String operator, Object right, boolean not) {
+        validateOperator(operator);
```

Comment:
For reference: This will partially address #14

Reply:
Why only partially? I also added a check for non-raw column names in `QueryColumn` in case you missed it (:

Reply:
With partially I only meant this code section. I did miss that one though. Thank you! I added the reference for this as well.